### PR TITLE
Add container build image commands

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #424

TODO:

- [x] Add remove command
- [ ] Add tests
- [ ] Update container capabilities doc section

### CODE REVIEW

File: pulpcore/cli/container/context.py [hunk 1]
```diff
@@ -143,6 +144,16 @@ def copy_manifest(
         body = self.preprocess_body(body)
         return self.call("copy_manifests", parameters={self.HREF: self.pulp_href}, body=body)
 
+    def build_image(
+        self,
+        container_artifact: str,
+        tag: Optional[str],
+        artifacts: Optional[str],
+    ) -> Any:
+        body = {"containerfile_artifact": container_artifact, "tag": tag, "artifacts": artifacts}
```
Comment (mdellweg) [inline pulpcore/cli/container/context.py:153]: I think, i would add a needs_capability here.

File: pulpcore/cli/container/context.py [hunk 2]
```diff
@@ -153,6 +164,10 @@ class PulpContainerPushRepositoryContext(PulpContainerBaseRepositoryContext):
         "roles": [PluginRequirement("container", "2.11.0")],
     }
 
+    def remove_image(self, digest: str) -> Any:
+        body = {"digest": digest}
```
Comment (mdellweg) [inline pulpcore/cli/container/context.py:168]: and here

**Comment by lubosmj** [comment]
@gerrod3, are you able to continue working on this PR? We fixed the issues with the builder on the plugin's side (https://github.com/pulp/pulp_container/commit/a6c2103cbf95739a23db6950e7ca32e0725401c7).

**Comment by gerrod3** [comment]
> @gerrod3, are you able to continue working on this PR? We fixed the issues with the builder on the plugin's side ([pulp/pulp_container@a6c2103](https://github.com/pulp/pulp_container/commit/a6c2103cbf95739a23db6950e7ca32e0725401c7)).

Yes, I can resume work on it now.

File: pulpcore/cli/container/repository.py [hunk 3]
```diff
@@ -291,6 +316,72 @@ def copy_manifest(
...
+    "--containerfile",
+    help=_(
+        "An artifact href of an uploaded Containerfile. Can also be a local Containerfile to be"
+        " uploaded using @."
+    ),
+    required=True,
+)
+@click.option("--tag", help=_("A tag name for the new image being built."))
+@click.option(
+    "--artifacts",
```
[the remark this answers is not in the document]
Comment (gerrod3) [reply pulpcore/cli/container/repository.py:345]: I haven't touched this PR is a while, so maybe it's best to close it.

**Pull request closed without merging**